### DISCUSSION TITLE
https://kdheepak.com/blog/neovim-languageserver-julia/

### DISCUSSION THREAD

**Discussion by giscus** [kdheepak/kdheepak.github.io#6] [Comments]
# https://kdheepak.com/blog/neovim-languageserver-julia/

https://kdheepak.com/blog/neovim-languageserver-julia/

**Comment by jd-foster** [comment]
Hi Dheepak, Thanks for this great guide. Having gone through your steps, I noticed a few updates are needed since your original post and might be useful for others later. These include:
1. Neovim is now at v0.6.1 or greater at time of writing, so should all work without needing to get the latest commit.
1. The `:LspInstall` command in Neovim has been deprecated, and shouldn't be needed if using your config script.
1. Line 13 in the .vimrc config should be `Plug 'neovim/nvim-lspconfig'`
1. Line 18 should be `    require'lspconfig'.julials.setup{}`
1. As an aside, I found out that the `.vimrc` configuration file for Neovim should go in `~/.config/nvim/init.vim`

Thanks again.

**Reply by kdheepak** [reply]
Thanks for sharing! I should make an update to this post or add a new post. I’m using nvim-cmp and nvim-lsp-installer these days, so my configuration has changed significantly.
